rc/tools/git.kak

```diff
@@ -2,10 +2,10 @@ declare-option -docstring "name of the client in which documentation is to be di
     str docsclient
 
 declare-option -docstring "git diff added character" \
-    str git_diff_add_char "▏"
+    str git_diff_add_char "▊"
```

Comment:
Yeah this is super hard to read, I can't even make out the green color. Good change.
Looks like the person who originally added this doesn't use it anymore..

I guess it's not as clear as `+` but that one is probably too easily confused with actual content.